assets/airports/rjss.json

```diff
@@ -0,0 +1,960 @@
+        }
+    },
+    "stars": {
+        "LANCEA": {
+            "icao": "LANCEA",
+            "name": "Lance East Alfa",
+            "entryPoints": {
+                "LANCE": [["LANCE", "A110+"]]
+            },
+            "body": [["FUBOU", "A60+"], ["SNOOK", "A40+"], ["TOPAZ", "S230-"], ["PERID", "A15+"]],
+            "rwy": {
+                "RJSS27": ["#360"]
+            },
+            "draw": [
+                ["LANCE*", "FUBOU", "SNOOK", "TOPAZ", "PERID"]
+            ]
+        },
+        "LANCEB": {
+            "icao": "LANCEB",
+            "name": "Lance East Bravo",
+            "entryPoints": {
+                "LANCE": [["LANCE", "A110+"]]
+            },
+            "body": [["FUBOU", "A60+"], ["SNOOK", "A40+"]],
+            "rwy": {
+                "RJSS27": ["#085"]
+            },
+            "draw": [
+                ["LANCE*", "FUBOU", "SNOOK"]
+            ]
+        },
+        "LANCEW": {
+            "icao": "LANCEW",
+            "name": "Lance West",
+            "entryPoints": {
+                "OWLET": [["LANCE", "A110+"]]
+            },
+            "body": [["QUAIL", "S250-"], ["RIBON", "A60+"]],
+            "rwy": {
+                "RJSS09": [["SHIPS", "A40+"]]
+            },
+            "draw": [
+                ["LANCE*", "QUAIL", "RIBON", "SHIPS"]
+            ]
+        },
+        "OWLEEA": {
+            "icao": "OWLEEA",
+            "name": "Owlet East Alfa",
+            "entryPoints": {
+                "OWLET": [["OWLET", "A130+"]]
+            },
+            "body": [["DATTE", "A80+"], ["RIBON", "A60+"], ["SNOOK", "A40+"], ["TOPAZ", "S230-"], ["PERID", "A15+"]],
+            "rwy": {
+                "RJSS27": ["#360"]
+            },
+            "draw": [
+                ["OWLET*", "DATTE", "RIBON", "SNOOK", "TOPAZ", "PERID"]
+            ]
+        },
+        "OWLEEB": {
+            "icao": "OWLEEB",
+            "name": "Owlet East Bravo",
+            "entryPoints": {
+                "OWLET": [["OWLET", "A130+"]]
+            },
+            "body": [["DATTE", "A80+"], ["RIBON", "A60+"], ["SNOOK", "A40+"]],
+            "rwy": {
+                "RJSS27": ["#067"]
+            },
+            "draw": [
+                ["OWLET*", "DATTE", "RIBON", "SNOOK"]
+            ]
+        },
+        "OWLETW": {
+            "icao": "OWLETW",
+            "name": "Owlet West",
+            "entryPoints": {
+                "OWLET": [["OWLET", "A130+"]]
+            },
+            "body": [["PRINK", "A130+"], ["QUIST", "S250-"], ["RIBON", "A60+"]],
+            "rwy": {
+                "RJSS09": [["SHIPS", "A40+"]]
+            },
+            "draw": [
+                ["OWLET*", "PRINK", "QUIST", "RIBON", "SHIPS"]
+            ]
+        },
+        "PERID": {
+            "icao": "PERID",
+            "name": "Perid",
+            "entryPoints": {
+                "AOBAR": ["AOBAR"]
+            },
+            "body": ["_SDE135010", "_SDE120010", "_SDE105010"],
```

Comment:
There is a discrepancy here due to magnetic deviation. Make sure to take it into account when calculating coordinates for invisible fixes. 
![image](https://user-images.githubusercontent.com/13125311/86006258-9c6dbc80-ba0d-11ea-912c-2da247f1ed64.png)

I now realise the same discrepancy can be found in the IXE8 and SDE6R departures where the radial is off by 8 degrees.

![image](https://user-images.githubusercontent.com/13125311/86006498-f53d5500-ba0d-11ea-8671-9d06ce36a176.png)

Reply:
So am I right to interpret that the name of the invisible fix will be kept the same (true heading), but the coordinate should be calculated (i.e. whatever I punch into GPS Visualizer or any calculator) based on magnetic heading?

Reply:
Correct 👍

Reply:
Thanks, fixed!